### PR TITLE
Sample code for the `UserString` vs `str` article

### PR DESCRIPTION
**Pull request by lpozo**
**Where to put new files:**

- New files should go into a top-level subfolder, named after the article slug. For example: `my-awesome-article`

**How to merge your changes:** 

1. [Make sure the CI code style tests all pass (+ run the automatic code formatter if necessary).](https://github.com/realpython/materials/blob/master/README.md)
2. Find an RP Team member on Slack and ask them to review & approve your PR.
3. Once the PR has one positive ("approved") review, GitHub lets you merge the PR.
4. 🎉

### CODE REVIEW

File: inherit-python-string/lower_string.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class LowerString(str):
+    def __init__(self, string):
+        super().__init__(string.lower())
```
Comment (acsany) [inline inherit-python-string/lower_string.py:3]: I think you can either remove this example or add a comment that it doesn't work. My vote would be for removing it.

File: inherit-python-string/mutable_string.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        self.data = self.data[: index - 1] + self.data[index:]
+
+    def upper(self):
+        self.data = self.data.upper()
+
+    def lower(self):
+        self.data = self.data.lower()
+
+    def sort(self, key=None, reverse=False):
+        self.data = "".join(sorted(self.data, key=key, reverse=reverse))
```
Comment (acsany) [inline inherit-python-string/mutable_string.py:18]: Please see my related comments in my review of the tutorials drafts repo 🤓

File: inherit-python-string/lower_string.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+
+# class LowerString(str):
+#     def __new__(cls, string):
+#         instance = super().__new__(cls, string.lower())
+#         return instance
+
```
Comment (acsany) [inline inherit-python-string/lower_string.py:10]: Add a blank line to be consistent with the other examples.

**Review by acsany — CHANGES_REQUESTED** [review]
Hi @lpozo, thanks for providing the materials for your article. I left you some minor comments, but I suggest that you wait with the updates and focus on the tutorial draft PR for now 🙂

File: inherit-python-string/lower_string.py (hunk 1 above)
Comment (acsany) [reply inherit-python-string/lower_string.py:3]: HI @lpozo, you didn’t make any changes here. I’m fine with keeping it if it’s your deliberate choice. I just want to double check that you didn’t forget about it 😉

Comment (lpozo) [reply inherit-python-string/lower_string.py:3]: Ohh, I missed this comment. I'll fix it now. Thanks!

**Review by acsany — APPROVED** [review]
Thanks for the additional update. I think you're all set for the DR 🥳

**Pull request merged**